tests/core/test_pod_security_policies.py

```diff
@@ -60,3 +75,66 @@ def test_service_accounts_have_role_binding(admin_mc, request):
             "default", kubernetes.client.V1DeleteOptions()))
 
     wait_for(lambda: service_account_has_role_binding(rbac, pspt), timeout=30)
+
+
+def test_pod_security_policy_template_del(admin_mc, request, remove_resource):
+    """ Test for pod security policy template binding correctly
+    ref https://github.com/rancher/rancher/issues/15728
+    ref https://localhost:8443/v3/podsecuritypolicytemplates
+    :param admin_mc:
+    :param request:
+    :return:
+    """
+    api_client = admin_mc.client
+
+    setup_cluster_with_pspt(api_client, request)
+    pspt_proj = api_client.create_pod_security_policy_template("pspt2")
+
+    #  create a project in order to establish bindings
+    #  shameless copy from test_rbac.py
+    def create_project():
+        try:
```

Comment:
should not try/catch around this. if any of this throws an error, it is a bug in the test or the code and the test should error out